packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,730 @@
+{{page-title "Alert component"}}
+
+<h2 class="dummy-h2">Alert</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">
+    An Alert is an element intended for
+    <strong>system-generated messages</strong>
+    (for messages that are the result of a user's actions see the
+    <LinkTo @route="components.toast">Toast</LinkTo>
+    component.)
+  </p>
```

Comment:
```suggestion
  <p class="dummy-paragraph">
    An Alert is an element intended for
    <strong>system-generated messages</strong>. It is a live region with important, usually time-sensitive information. The use of this alert component will cause immediate notifications for users with assistive technology. Since alerts are not required to receive focus, it should not be required that the user close the alert.<p>
    <p class="dummy-paragraph">For messages that are the result of a user's actions see the
    <LinkTo @route="components.toast">Toast</LinkTo>
    component.)
  </p>
```